src/it/scala/org/apache/spark/sql/snowflake/SFTestWrapperSparkSession.scala

```diff
@@ -28,6 +28,10 @@ class SFTestWrapperSparkSession(sc: SparkContext, sfConfigs: Map[String, String]
       .options(sfConfigs)
       .option("dbtable", tableName)
       .save()
+    if (df.count() == 0) {
```

Comment:
Why do you need this ?

Reply:
That's weird. Also what you have here is extremely inefficient. You are evaluating the dataframe twice. Instead just do a create if not exists on the table.
Is th efact that a table is not created a bug in our connector ?

Reply:
`df.write` in `Overwrite` mode should still create the table with zero rows, I think. If it doesn't, that is a bug.